python/ray/autoscaler/v2/instance_manager/subscribers/threaded_ray_installer.py

```diff
@@ -31,65 +46,50 @@ def __init__(
         self._max_concurrent_installs = max_concurrent_installs
         self._max_install_attempts = max_install_attempts
         self._install_retry_interval = install_retry_interval
+        self._error_queue = error_queue
         self._ray_installation_executor = ThreadPoolExecutor(
             max_workers=self._max_concurrent_installs
         )
 
     def notify(self, events: List[InstanceUpdateEvent]) -> None:
         for event in events:
-            if event.new_instance_status == Instance.ALLOCATED:
+            if event.new_instance_status == Instance.RAY_INSTALLING:
```

Comment:
ThreadedRayInstaller should install Ray after the instance status has been changed to `RAY_INSTALLING` by the autoscaler v2 reconciler; therefore, we filter `RAY_INSTALLING` here.